Makefile

```diff
@@ -2328,10 +2328,23 @@ else
 endif
 
 .PHONY: nncp_with_retries
-nncp_with_retries: ## Deploy NNCP with retries
+nncp_with_retries: nncp_dns ## Deploy NNCP with retries
 	 $(eval $(call vars,$@))
 	bash scripts/retry_make_nncp.sh $(NNCP_RETRIES)
 
+.PHONY: nncp_dns
+nncp_dns: export DNS_SERVER=${NNCP_DNS_SERVER}
+nncp_dns:
+       $(eval $(call vars,$@,nncp))
```

Comment:
I believe the indentation needs to be done with tabs instead of spaces

Reply:
yes, indeed. Good catch! Done.